sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -140,6 +140,8 @@ export interface AppendBlobAppendBlockFromURLOptions extends CommonOptions {
     conditions?: AppendBlobRequestConditions;
     customerProvidedKey?: CpkInfo;
     encryptionScope?: string;
+    // Warning: (ae-forgotten-export) The symbol "HttpAuthorization" needs to be exported by the entry point index.d.ts
+    sourceAuthorization?: HttpAuthorization;
```

Comment:
can you please export all missing definitions?

Reply:
Resolved.